libs/api/domains/aircraft-registry/src/lib/models/aircraft.model.ts

```diff
@@ -0,0 +1,61 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType('AircraftRegistryPerson')
+class Person {
+  @Field(() => String, { nullable: true })
+  address?: string | null
+
+  @Field(() => String, { nullable: true })
+  city?: string | null
+
+  @Field(() => String, { nullable: true })
+  country?: string | null
+
+  @Field(() => String, { nullable: true })
+  email?: string | null
+
+  @Field(() => String, { nullable: true })
+  name?: string | null
+
+  @Field(() => String, { nullable: true })
+  phone?: string | null
+
+  @Field(() => String, { nullable: true })
+  postcode?: string | null
+
+  @Field(() => Number, { nullable: true })
+  ssn?: number | null
+}
+
+@ObjectType('AircraftRegistryAircraft')
+export class Aircraft {
+  @Field(() => String, { nullable: true })
```

Comment:
Should all of these fields be nullable?

Reply:
Since the openapi schema suggests that these values can be null, I basically went overboard and typed everything as nullable 🙃